decode_morse.rb

```diff
@@ -0,0 +1,54 @@
+@morse_chat = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}
+
+def decode_char(char)
+  @current_char = @morse_chat[char]
+  @current_char
+end
+
+def decode_word(word)
+  @current_word = word.split
+  @decoded_word = ''
+
+  @current_word.each do |char|
+    decode_char(char)
+    @decoded_word += decode_char(char)
+  end
+end
+
+def decode_message(message)
+  @words = message.split('   ')
+  @words.each do |word|
+    decode_word(word)
+  end
+end
```

Comment:
- [ ] These functions do not return the decode messages as intended. Kindly fix that.

  **SUGGESTION:**
  - In **`decode_word`** method consider returning `@decoded_word` at the end of the method.
  - In **`decode_message`** method consider defining, updating and returning a variable as you have done with `@decoded_word` earlier. 

  ![some-th](https://user-images.githubusercontent.com/36542986/160815742-68d3a8dc-74a7-4e12-b38f-02ce2e281aff.jpg)